packages/editor-ui/src/components/CommunityPlusEnrollmentModal.test.ts

```diff
@@ -0,0 +1,144 @@
+import { createTestingPinia } from '@pinia/testing';
+import userEvent from '@testing-library/user-event';
+import { createComponentRenderer } from '@/__tests__/render';
+import CommunityPlusEnrollmentModal from '@/components/CommunityPlusEnrollmentModal.vue';
+import { COMMUNITY_PLUS_ENROLLMENT_MODAL } from '@/constants';
+import { useUsageStore } from '@/stores/usage.store';
+import { useToast } from '@/composables/useToast';
+import { useTelemetry } from '@/composables/useTelemetry';
+
+vi.mock('@/composables/useToast', () => {
+	const showMessage = vi.fn();
+	const showError = vi.fn();
+	return {
+		useToast: () => {
+			return {
+				showMessage,
+				showError,
+			};
+		},
+	};
+});
+
+vi.mock('@/composables/useTelemetry', () => {
+	const track = vi.fn();
+	return {
+		useTelemetry: () => {
+			return {
+				track,
+			};
+		},
+	};
+});
+
+const renderComponent = createComponentRenderer(CommunityPlusEnrollmentModal, {
+	global: {
+		stubs: {
+			Modal: {
+				template:
+					'<div role="dialog"><slot name="header" /><slot name="content" /><slot name="footer" /></div>',
+			},
+		},
+	},
+});
+
+describe('CommunityPlusEnrollmentModal', () => {
+	const buttonLabel = 'Send me a free license key';
+
+	beforeEach(() => {
+		createTestingPinia();
+	});
+
+	it('should test enrolling', async () => {
+		const closeCallbackSpy = vi.fn();
+		const usageStore = useUsageStore();
+		const registerCommunityEditionSpy = vi
```

Comment:
would you mind using mockedStore from `packages/editor-ui/src/__tests__/utils.ts`

Reply:
sure, done